### DISCUSSION TITLE
How to work with pytest-listener

### DISCUSSION THREAD

**Discussion by HasiniRangana** [pytest-dev/pytest#11130] [Q&A]
I want to know how to work with pytest-listener.
If there is any documentation or anything to follow please share with me.
How can we implement tests using this plugin after installing it?

**Comment by RonnyPfannschmidt** [comment]
its unclear what you are asking for - pytest-listener is not part of pytest or pytest-dev - it seems undocumented

**Reply by HasiniRangana** [reply]
I want to know how to work with this.
https://pypi.org/project/pytest-listener

**Reply by RonnyPfannschmidt** [reply]
And as far as I am aware nobody in pytest-dev knows,this is a 3rd party Projekt
